docs/proposals/networking.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package networking;
+
+message CreateInterfaceRequest {
+    string ip_address = 1;
+    repeated int32 ports = 2;
+}
+
+message CreateInterfaceResponse {
+    string interface_name = 1;
+}
+
+message DeleteInterfaceRequest {
+    string interface_name = 1;
+}
+
+message CreateContainerNetworkInterfaceRequest {
+    string ip_address = 1;
+    string sub_network = 2;
+}
+
+message CreateContainerNetworkInterfaceResponse {
+    string interface_name = 1;
+}
+
+message Empty {}
+
+service Networking {
+    rpc CreateInterface(CreateInterfaceRequest) returns (CreateInterfaceResponse) {}
+    rpc DeleteInterface(DeleteInterfaceRequest) returns (Empty) {}
```

Comment:
A `CreateContainerNetworkInterface` rpc is missing

Reply:
Indeed, fixed that in my latest commit !